src/SSW.SophieBot/components/SSWSophieBot.HttpClientComponents.PersonQuery/EmployeesHelper.cs

```diff
@@ -137,6 +137,16 @@ public static GetAppointmentModel GetAppointmentBy(DateTime date, List<GetAppoin
             return results.Count != 0 ? results[0] : null;
         }
 
+        public static List<string> GetClientsBy(DateTime date, List<GetAppointmentModel> appointments)
+        {
+            var clientAppointments = appointments
+                .Where(appointment => date.Date.Ticks >= GetTicksFrom(appointment.Start) && date.Date.Ticks <= GetTicksFrom(appointment.End))
```

Comment:
```suggestion
                .Where(appointment => date.Date.Ticks >= GetTicksFrom(appointment.Start) && date.Date.Ticks < GetTicksFrom(appointment.End.AddDays(1)))
```
I think we should use this logic here. Do you agree?